core/src/main/java/org/jboss/pnc/rex/core/config/RequestRetryPolicy.java

```diff
@@ -20,6 +20,6 @@
 import io.smallrye.config.ConfigMapping;
 import org.jboss.pnc.rex.core.config.api.MutinyRetryPolicy;
 
-@ConfigMapping(prefix = "scheduler.options.http-configuration.425-backoff-policy")
-public interface Backoff425Policy extends MutinyRetryPolicy {
+@ConfigMapping
```

Comment:
I guess this is actually the previous HttpRetryPolicy.class. Could you re-add the prefix?
```suggestion
@ConfigMapping(prefix = "scheduler.options.http-configuration.request-retry-policy")
```

Reply:
@matejonnet  This serves 2 purposes, IDE navigation so that IntelliJ can recognize keys in **application.yaml** and redirect you to a class and for CDI `@Inject`, so please re-add it.

Reply:
Ok, but what to you mean by CDI inject ? Inject should work regardless of the string preffix being there or not ?

Reply:
No it doesn't work. If you'd want to **directly** inject this part of configuration, it wouldn't run. F.e. if you remove the prefix or the annotation from HttpConfiguration and run tests, it will fail in GenericVertxHttpClient and won't be able to find it.

It's a convention of mine to have this in all configurations, so that if someone copy-pastas, he'd get immediate option to inject it by CDI.